xml/System.Numerics.Tensors/TensorSpan`1.xml

```diff
@@ -978,7 +1002,7 @@
       <Docs>
         <param name="left">To be added.</param>
         <param name="right">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Returns <see langword="true" /> if left and right point at the same memory and have the same length.  Note that this does *not* check to see if the *contents* are equal.</summary>
```

Comment:
```suggestion
        <summary>Compares two <see cref="System.Numerics.Tensors.TensorSpan&lt;T&gt;" /> structs for equality.</summary>
```

Reply:
I'll add the prefix `T:`, and I thought the typeparam had to be `{T}`, not `<T>`. Am I wrong?